content/v2.0/reference/flux/stdlib/experimental/geo/filterrows.md

```diff
@@ -0,0 +1,176 @@
+---
+title: geo.filterRows() function
+description: >
+  The geo.filterRows() function filters data by a specified geographic region with
+  the option of strict filtering.
+menu:
+  v2_0_ref:
+    name: geo.filterRows
+    parent: Geo
+weight: 301
+v2.0/tags: [functions, geo]
+related:
+  - /v2.0/reference/flux/stdlib/experimental/geo/gridfilter/
+  - /v2.0/reference/flux/stdlib/experimental/geo/strictfilter/
+---
+
+The `geo.filterRows()` function filters data by a specified geographic region with
+the option of strict filtering.
+This function is a combination of [`geo.gridFilter()`](/v2.0/reference/flux/stdlib/experimental/geo/gridfilter/)
+and [`geo.strictFilter()`](/v2.0/reference/flux/stdlib/experimental/geo/strictfilter/).
+
+_**Function type:** Transformation_
+
+```js
+import "experimental/geo"
+
+geo.filterRows(
+  region: {lat: 40.69335938, lon: -73.30078125, radius: 20.0}
+  minSize: 24,
+  maxSize: -1,
+  level: -1,
+  s2cellIDLevel: -1,
+  correlationKey: ["_time"],
+  strict: true
+)
+```
+
+## Parameters
+
+The region containing the desired data points.
+_See [Region definitions](/v2.0/reference/flux/stdlib/experimental/geo/#region-definitions)._
+
+_**Data type:** Object_
+
+### minSize
+Minimum number of tiles that cover the specified region.
+Default is `24`.
+
+_**Data type:** Integer_
+
+### maxSize
+Maximum number of tiles that cover the specified region.
+Default is `-1`.
+
+_**Data type:** Integer_
+
+### level
+Desired [cell level](https://s2geometry.io/resources/s2cell_statistics.html) of grid tiles.
+Default is `-1`.
```

Comment:
didn't see -1 on the stats page...is that intentional?

Reply:
Yes, -1 means the level isn't specified.